sequins.go

```diff
@@ -160,12 +160,8 @@ func (s *sequins) init() error {
 		for range sighups {
 			// Refresh any feature flags on HUP
 			if s.goforit != nil {
-				err := goforit.RefreshFlags(s.goforit)
-				if err != nil {
-					log.Printf("Error refreshing flags on HUP: %s", err)
```

Comment:
ooc, why remove the log?

Reply:
We no longer have an error return from RefreshFlags, that's all internal now :)